dataland-frontend/src/components/resources/frameworkDataSearch/p2p/P2pDataModel.ts

```diff
@@ -238,8 +238,8 @@ export const p2pDataModel = [
             name: "absoluteEmissions",
             label: "Absolute emissions",
             description:
-              "Transparent accounting & disclosure of all greenhouse gas emissions (Scope 1, Scope 2, Scope 3) along the following metric: absolute emissions (t CO2e)",
-            unit: "",
+              "Transparent accounting & disclosure of all greenhouse gas emissions (Scope 1, Scope 2, Scope 3) along the following metric: absolute emissions",
```

Comment:
In the upload page the unit to be provided is now no longer visible. Please check with the authors of DALA-2373 if this is okay and if the unit will become visible again if DALA-2373 is merged in.